apps/mark-scan/accessible-controller/src/controllerd.rs

```diff
@@ -28,10 +30,21 @@ mod device;
 mod port;
 
 const APP_NAME: &str = "vx-mark-scan-controller-daemon";
+const KPB_200_FW_VID: u16 = 10445;
+const KPB_200_FW_PID: u16 = 16392;
```

Comment:
These are frequently displayed as hex values rather than decimal values (e.g. in the output of `lsusb` and often in device documentation). Consider using hex?
```suggestion
const KPB_200_FW_VID: u16 = 0x28cd;
const KPB_200_FW_PID: u16 = 0x4008;
```